pkg/testrunner/result/output.go

```diff
@@ -45,6 +47,10 @@ func Output(config *Config, tmClient kubernetes.Interface, namespace string, tr
 
 	metadata.TestrunID = tr.Name
 
+	if _, err := url.ParseRequestURI(config.ArgoUIEndpoint); err != nil {
```

Comment:
Shouldn't this be...? Not sure. Also the error message isn't handled. I'm missing something like: if err != nil { logger.log("Can't parse ArgoUIEndpoint")...
```suggestion
	if _, err := url.ParseRequestURI(config.ArgoUIEndpoint); err == nil {
```

Reply:
I wanted to use this check just to verify if the url is set and if it is a valid url.
If not then the field should just be ignored and only added in case of no error

Reply:
done